feature-libs/cart/main/root/components/mini-cart/mini-cart.component.ts

```diff
@@ -0,0 +1,22 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { ICON_TYPE } from '@spartacus/storefront';
+import { Observable } from 'rxjs';
+import { MiniCartComponentService } from './mini-cart-component.service';
+
+@Component({
+  selector: 'cx-mini-cart',
+  templateUrl: './mini-cart.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class MiniCartComponent {
+  iconTypes = ICON_TYPE;
+
+  quantity$: Observable<number>;
```

Comment:
`quantity$ = this.miniCartComponentService.getQuantity();`, just set it in the component, no need do it in `constructor` (same for `total$`).

Reply:
fixed